frontend/components/navigation/Navbar.tsx

```diff
@@ -13,53 +14,57 @@ const inter = Inter({
 });
 
 interface NavbarProps {
-	/**
-	 * show login or logout button
-	 */
-	login?: boolean;
 	/**
 	 * Additional classes
 	 */
 	className?: string;
-	/**
-	 * alternative click handler
-	 */
-	onClick?: () => void;
 }
 
 /**
  * UI component for diplaying the Navbar
  */
-export const Navbar = ({ login, className = "", onClick }: NavbarProps) => {
+export const Navbar = ({ className = "" }: NavbarProps) => {
+	const router = useRouter();
+	const [loggedIn, setLoggedIn] = useState<boolean>();
+	useEffect(() => {
+		if (router.pathname == "/login") {
+			setLoggedIn(undefined);
+		} else {
+			setLoggedIn(hasCookie("access_token"));
+		}
+	});
 	return (
-		<div
+		<nav
 			className={`flex items-center justify-between p-5 md:p-10 ${className}`}
 		>
-			<Logo onClick={onClick}></Logo>
-			{login ? (
-				<div className="flex flex-row">
-					<LogOut
-						className="text-kiokuDarkBlue hover:cursor-pointer"
-						onClick={async () => {
-							const response = await authedFetch("/api/logout", {
-								method: "POST",
-							});
-							if (response?.ok) {
-								location.reload();
-							}
-						}}
-					></LogOut>
-				</div>
-			) : (
+			<Logo
+				onClick={() =>
+					loggedIn ? router.push("/") : router.push("/home")
+				}
+			></Logo>
+			{loggedIn == true && (
+				<LogOut
+					className="text-kiokuDarkBlue hover:cursor-pointer"
+					onClick={async () => {
+						const response = await authedFetch("/api/logout", {
+							method: "POST",
+						});
+						if (response?.ok) {
+							router.replace("/home");
+						}
+					}}
+				></LogOut>
+			)}
+			{loggedIn == false && (
 				<Button
 					id="loginButton"
 					style="secondary"
 					className="invisible h-full justify-end sm:visible"
```

Comment:
```suggestion
					className="invisible justify-end sm:visible h-7 md:h-10"
```
The Login Button is causing a shift in the position of the Logo when changing from `home` to `login` page.

Reply:
changed in 8c37524